### DISCUSSION TITLE
How to integrate flowbit with React?

### DISCUSSION THREAD

**Discussion by kunal-dd** [themesberg/flowbite#164] [Q&A] [answered]
Hello,

I've followed all the steps mentioned in the documentation (https://flowbite.com/docs/getting-started/react) but still not getting flowbite work.

Imported flowbite in **index.js**

![image](https://user-images.githubusercontent.com/34878128/170532581-80dcaf71-4ce5-4123-86be-c84cc89c5c86.png)

Changed the **tailwind.config.js** as below,

![image](https://user-images.githubusercontent.com/34878128/170532744-324fba31-8ad1-4895-a99e-631ac71a68b1.png)

Please let me know if I'm missing anything here.

Thanks.

**Comment by zoltanszogyenyi** [comment, marked as answer]
Please check out the dedicated React component library:

https://github.com/themesberg/flowbite-react
